Sources/Classes/core/Models/MiniAppManifest.swift

```diff
@@ -183,11 +183,13 @@ public struct MiniAppManifest: Codable, Equatable {
              versionId
     }
 
-    init(requiredPermissions: [MASDKCustomPermissionModel]?,
-         optionalPermissions: [MASDKCustomPermissionModel]?,
-         customMetaData: [String: String]?,
-         accessTokenPermissions: [MASDKAccessTokenScopes]?,
-         versionId: String?) {
+    public init(
```

Comment:
Yeah the struct is already `public` and conforms to `Codable` so imho the initializer can be public too because it's accessible to same way by parsing a json into `MiniAppInfo`.
It makes it more convenient to initialize for tests and eg. one case in `MiniAppTermsView` to be able to preview different kinds of infos.
wdyt?